playground/python/Python.AppHost/Program.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Python;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddPythonProjectWithVirtualEnvironment("script-only", "../script_only", "main.py");
```

Comment:
This is a pretty long `Add...` method. Can we make it more concise?

Reply:
I can remove the virtual environment part and make that implied. I don't think anyone wants to work on python projects without a virtual env.

Reply:
Make it a bool? Add a doc comment?